pkg/quarterdeck/db/models/users.go

```diff
@@ -302,6 +303,97 @@ func (u *User) UpdateLastLogin(ctx context.Context) (err error) {
 	return tx.Commit()
 }
 
+const (
+	getUserSQL = "SELECT id, name, email, terms_agreement, privacy_agreement, last_login, created, modified from users where id in (select user_id FROM organization_users where organization_id = $1)"
+)
+
+// ListUsers returns a paginated collection of users filtered by the orgID.
+// The orgID must be a valid non-zero value of type ulid.ULID.
```

Comment:
I think it could also be a string or slice of bytes, so long as it can be parsed by `ulids.Parse`

Reply:
Ok, just checked the tests for `Parse` -- here's what it has for [some examples](https://github.com/rotationalio/ensign/blob/main/pkg/utils/ulid/ulid_test.go#L42-L66) of vals that are valid vs invalid: